bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/VolvoOnCallHandlerFactory.java

```diff
@@ -93,13 +93,4 @@ private void unregisterDeviceDiscoveryService(ThingUID thingUID) {
             discoveryServiceRegs.remove(thingUID);
         }
     }
-
-    @Reference
-    protected void setDynamicStateDescriptionProvider(VehicleStateDescriptionProvider provider) {
-        this.stateDescriptionProvider = provider;
-    }
-
-    protected void unsetDynamicStateDescriptionProvider(VehicleStateDescriptionProvider provider) {
-        this.stateDescriptionProvider = null;
-    }
```

Comment:
This file isn't part of this binding, please remove this change.

Reply:
Oops, that was done on a previous commit by mistake.
Commit hash:
```
$ git log VolvoOnCallHandlerFactory.java 
commit 84c0d48c8175a5a5faee15f5b26505935a04db29
Author: jannegpriv <jannegpriv@gmail.com>
Date:   Sun Jan 26 00:28:20 2020 +0100
```
What is the easiest way to do that?

Reply:
The easiest way would be to just add those removed lines of code back in and commit it to this PR.

Reply:
Fixed!